slack.md

```diff
@@ -26,16 +26,39 @@ title: Slack
 In addition, [numerous other community resources](./download.html) are
 available for help.
 
-## Channels
+## User-related channels
 
-* [#announcements](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) - announcements about Trino
-* [#beginner](https://{{site.slack_fqdn}}/messages/C0305TQ05KL) - posts when you don't know where to ask a question
+* [#announcements](https://{{site.slack_fqdn}}/messages/CFLB9AMBN) -
+  announcements about new releases, events, and more
+* [#beginner](https://{{site.slack_fqdn}}/messages/C0305TQ05KL) - post when you
+  don't know where to ask a question
 * [#troubleshooting](https://{{site.slack_fqdn}}/messages/CGB0QHWSW) - need help?
-* [#dev](https://{{site.slack_fqdn}}/archives/CP1MUNEUX) - discussions about Trino codebase, development, and developing extensions
-* [#triage](https://{{site.slack_fqdn}}/messages/CFP4810CV) - discussions and coordination about new issues and PRs from the community
-* [#releases](https://{{site.slack_fqdn}}/messages/CFP480UKX) - notices about upcoming releases
-* [#github](https://{{site.slack_fqdn}}/messages/CFN644R4G) - feed of activity on [GitHub]({{site.github_org_url}})
-* [#documentation](https://{{site.slack_fqdn}}/messages/C01TEP0HJTH) - discuss [Trino documentation]({{site.url}}/docs/current/)
+* [#kubernetes](https://{{site.slack_fqdn}}/messages/CFSDUNJ4W) - container
+  images and deployment with the Helm chart on Kubernetes
+* [#python-client](https://{{site.slack_fqdn}}/messages/CFPVDCDHV) - anything
+  related to Python-based client applications and tools
+* [#documentation](https://{{site.slack_fqdn}}/messages/C01TEP0HJTH) - discuss
+  [Trino documentation]({{site.url}}/docs/current/)
+
+## Language-specific channels
+
+A number of channels are available for users who prefer to communicate in their
+respective language, or to discuss regional topics around Trino.
+
+* [#general-cn](https://{{site.slack_fqdn}}/messages/CKS1YGP27) - Chinese
+* [#general-de](https://{{site.slack_fqdn}}/messages/C047U4F5D7F) - German
+* [#general-jp](https://{{site.slack_fqdn}}/messages/CLEDR9V7G) - Japanese
+
+## Trino development channels
+
+* [#dev](https://{{site.slack_fqdn}}/archives/CP1MUNEUX) - discussions about
+  Trino codebase, development, and developing extensions
+* [#triage](https://{{site.slack_fqdn}}/messages/CFP4810CV) - discussions and
+  coordination about new issues and PRs from the community
+* [#github](https://{{site.slack_fqdn}}/messages/CFN644R4G) - feed of activity
+  on [GitHub]({{site.github_org_url}})
+* [#releases](https://{{site.slack_fqdn}}/messages/CFP480UKX) - notices about
+  upcoming releases
```

Comment:
Should we add specific ones like iceberg and hudi here?

Reply:
I suggested tech our connector specific channels earlier and it was decided no since these come and go. I would say no and stuck with the general dev and troubleshooting and then the user channels as you suggest.

Reply:
Fair enough